src/__tests__/renderIntoDocument.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Due to issues with operating systems, filename case sensitivity, and git, I prefer all my files to use `kebab-casing` rather than `camelCasing`. Could you rename this file please? I really should have an eslint rule for it.